qaframework-bdd-tests/cypress.json

```diff
@@ -5,7 +5,7 @@
     "**/*.feature"
   ],
   "video": true,
-  "defaultCommandTimeout": 20000,
+  "defaultCommandTimeout": 120000,
```

Comment:
Is it possible to have one single default time out or it keep fluctuating as the server grows!

Reply:
The patient dashboard takes 110-115 seconds to load. So I think we should keep the timeout at 120 seconds for now until it is fixed. 
Currently, @kumuditha-udayanga is testing the possibility of running the tests against a docker environment. Completing that will fix the timeout issue also.